packages/react/react/src/components/Portal/Portal.en.md

```diff
@@ -0,0 +1,77 @@
+# Portal
+
+The Portal component uses `React.createPortal` to render the child component on a DOM node outside of the parent component's DOM hierarchy.
+
+The Portal Component renders a Portal Node in `document.body` by default. However, if you pass in the Portal Component's `containerRef` prop, you can render the Portal Node in a `different DOM Node` than `document.body`.
+
+Additionally, it supports `nested portal functionality`. Nesting multiple portal components creates `a nested portal DOM hierarchy`.
+
+The Portal component is ideal for use with features like `Modal`, `Dialog`, and `Tooltip`.
+
+[React CreatePortal](https://react.dev/reference/react-dom/createPortal#rendering-a-modal-dialog-with-a-portal) Please refer to the following articles
+
+<br />
+
+```tsx
+function Portal({
+  children,
+  containerRef,
+}: {
+  children: React.ReactNode;=
```

Comment:
```suggestion
  children: React.ReactNode;
```

Reply:
@ChanhyukPark-Tech 
OMG.. Thank you. I'll fix it right now

Reply:
You can easily fix it just by pressing the `commit suggestion` button on my comment!

Reply:
@ChanhyukPark-Tech 
.....😂😂😂😂😂😂😂😂😂
Thank you so much for these tip. I was being an idiot and modifying the code myself every time, registering commits one by one.